sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -316,19 +316,34 @@ You can also manually refresh the certificate by calling this method:
 KeyVaultCertificates.refreshCertsInfo();
 ```
 
-### Side-loading certificates
+### Specific path certificates
+This starter also allows you to load the certificate in the specific path as
+trusted certificates if following property is configured.
```

Comment:
AzureKeyVault keystore will load certificates in the specific path:
1. well-know path: /etc/certs/well-known/
2. custom path: /etc/certs/custom/

The 2 paths can be configured by these propreties:
...